.github/workflows/lint.yml

```diff
@@ -0,0 +1,35 @@
+name: Lint
```

Comment:
Does this file need the copyright header?

Reply:
Nope (it doesn't in other repos)